carapace-server/src/main/java/org/carapaceproxy/server/certificates/ocsp/Digester.java

```diff
@@ -0,0 +1,84 @@
+/*
+ Licensed to Diennea S.r.l. under one
+ or more contributor license agreements. See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership. Diennea S.r.l. licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+ This code has been inspired by The Netty Project https://github.com/netty/netty
```

Comment:
we have to write this into a NOTICE file on the root of this git repository

This software contains code regarding OCSP copied from The Netty Project https://github.com/netty/netty